spec/services/token_manager_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+class Object
+  def headers
+    {"Authorization": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.
+      eyJ1c2VyIjoxLCJleHAiOjE0NTc5NjEyMzh9.
+      PTyASPDQ2BzMmktgnc57YXHEVPPiKS5lEg9xz2CgsUQMmtbptF3tnhvJ7YSLG3oX1HClKh"}
+  end
+end
```

Comment:
Getting the token is not an issue, getting an object with `headers` method is the reason for this.